### ISSUE TITLE
migrate google analytics to v4

### ISSUE THREAD

**Issue by halkeye** [jenkins-infra/helpdesk#3530]
### Service(s)

Other

### Summary

when you go into https://analytics.google.com/

```
On July 1, 2023, this property will stop processing data. Starting in March 2023, for continued website measurement, you should create a new Google Analytics 4 (GA4) property, or one will be created for you based on your original property and reusing existing site tags.
```

When I click that link i get so someone with more permissions should go through it.

![image](https://user-images.githubusercontent.com/110087/233209115-02745532-b1fd-4bc6-afe6-e8960d5f1d13.png)

~Or we do a full migration to matomo~

**Comment by timja** [comment]
I don't believe I have GA access, probably @olblak @rtyler or @oleg-nenashev

**Comment by olblak** [comment]
I granted administrator access to @dduportal  and viewer access to @MarkEWaite  and @timja 
@halkeye also has viewer access

Current administrator are KK, Tyler, @dduportal and myself

**Comment by dduportal** [comment]
> I granted administrator access to @dduportal and viewer access to @MarkEWaite and @timja @halkeye also has viewer access
> 
> Current administrator are KK, Tyler, @dduportal and myself

Thanks @olblak However, you'vre granted me administration access to the analytics property itself, but not to the account: I have the same message as Gavin (You do not have the permission, contact the Admin account).

**Comment by halkeye** [comment]
@dduportal how do you feel about just moving everything to matomo? We get better data privacy. I can move the [matamo](http://jenkins-matomo.do.g4v.dev/) (we use for plugins/stories) to the k8s cluster or something (instead of my droplet). Then we don't care what google does. We can give out more access, infact I already have anonymous access given out to give basic data.

**Comment by halkeye** [comment]
> [@halkeye](https://github.com/halkeye) could you describe, in [jenkins-infra/helpdesk#3530](https://github.com/jenkins-infra/helpdesk/issues/3530) or here, a summary of matomo: we're not against the fact of using it but we need to now what kind of service and requirements it has:

https://matomo.org/faq/on-premise/matomo-requirements/

My 2 GB Memory / 2 Intel vCPUs / 60 GB Disk droplet has been running matomo for plugins and stories for a couple years now without noticing anything at all.

> Storage (to allow us anticipate the storage policy and backups) or any persistence storage

https://matomo.org/faq/on-premise/managing-your-databases-size/

I actually can't find the hits x size page i found earlier. I'll attach the current database report, but it looks like so far in about 2 years we've used like 6gb of sql data. (I did trigger data cleanup after 90 days last week though, i didn't realize we were storing everything forever). I think 3 years of data was like 30 gb?

I built the docker image so everything is in the container, no runtime disk persistence needed

> Web service (are there different exposed services so we can check the DNS names/requirements for certificates and loadbalancing)

Just the PHP app

![Screenshot 2023-06-04 at 09-51-05 Database usage - Administration - Matomo](https://github.com/jenkins-infra/helpdesk/assets/110087/ac13d18c-6d55-4c05-8ba6-bee10cef87e2)

**Comment by halkeye** [comment]
They have integration with oauth and [ldap](https://plugins.matomo.org/LoginLdap#description) if want single login
